CONTRIBUTING.md

```diff
@@ -6,9 +6,15 @@
 * [.NET Core SDK](https://www.microsoft.com/net/learn/get-started/windows)
     * Version driven by [global.json](src/Common/Commands.Common/global.json)
     * Cmdlets are designed to use .NET Core 2.0 SDK with .NET Standard 2.0
+* [Visual Studio Code](https://code.visualstudio.com/download)
+    * Experimental and optional, still need Visual Studio installed
```

Comment:
You PR description says "or .NET Framework SDK". Is that the case? You may want to mention that here since that should be a lower barrier to entry.

Reply:
Looking at it more you still need VS because that is how I locate .NET Framework MSBuild.